wiki/Tournaments/OMIC/2019_4K/en.md

```diff
@@ -2,9 +2,415 @@
+- All players are advised to **have all maps from the mappool ready** before the match starts.
+- If there is no referee during the match time, players are expected to inform the **organizer staff** in due time.
+- Participants are expected to **show up on time** according to the schedule.
+- The staff can only give a delay of **10 minutes**.
+- If a player has a connection problem which caused them to be disconnected from the match, then the decision will be **determined by the referee** which is **absolute**.
+- Players must follow the [rules](/wiki/Rules) that has been set by the game.
+
+### Match Regulations 
+
+- All players who are deemed eligible for the tournament will enter the Qualifiers round.
+- Each lobby contains a maximum of 8 players in the Qualifiers round.
+- Players are allowed to use visual mods such as (HD), Flashlight (FL), Fade In (FI), and Mirror (MR).
+- Players will play 2 maps for a warm-up session that had been selected by the player that rolled the highest.
+- **Qualifiers round**
+  - The maximum number of players to enter the next round is 56 players.
+  - In the Qualifiers round there will be 4 maps in the mappool that have different skillsets, all of these maps will be selected by the mappool selector.
+  - All players will have 2 chances to play in each map, which means a total of 8 songs played.
+  - The highest score played for each beatmap will be counted and the Qualifiers score will be accumulated based on the comparison of scores of other players.
+  - After the Qualifiers round ends there will be a distribution of players based on seed.
+  - The function of seed is for the distribution of the next round, where players are randomized based on different seed.
+  - Seed distribution is based on the player's position in the Qualifiers:
+    - 1 - 16: Top Seed
+    - 17 - 32: High Seed
+    - 33 - 48: Medium Seed
+    - 49 - 56: Low Seed
+    - Players who get the Top Seed will immediately enter the Group Distribution Stage, without having to go to the Round of 56 first.
+- **Round of 56**
+  - In this round, there will be 8 groups with **2 high seed, 2 medium seed, and 1 low seed** in each group.
+  - The players who will qualify for the next round are the players who are in the top **2 positions** in each group.
+  - In the Round of 56 there will be **7 mappools (with 1 Tiebreaker)**.
+  - Each player collects points by reaching **positions 1 to 3**.
+  - The scoring system is as follows:
+    - 1st: 3 Point
+    - 2nd: 2 Point
+    - 3rd: 1 Point
+    - If both players get the same score in the 1-2 position, then the points will be taken from the top position. For example :
+      - Player A's score: 997.880 (3 Poin)
+      - **Player B's score: 995.522 (2 Point)**
+      - **Player C's score: 995.522 (2 Point)**
+      - Player D's score: 994.695 (0 Point)
+    - The Tiebreaker map will be used when players with **the second position and below have the same points** to get the 2nd place.
```

Comment:
this rule kinda overlaps with the previous rule, do you mean *3rd place and below*?

```suggestion
    - The tiebreaker map will be used when players with **the second position and below have the same points** to get the 2nd place.
```